### PR TITLE
Time substitutions should be in org timezone

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/flows/models.py [hunk 1]
```diff
@@ -1185,11 +1185,12 @@ def build_message_context(self, contact, msg):
         flow_context = dict()
 
         date_format = get_datetime_format(self.org.get_dayfirst())[1]
+        tz = pytz.timezone(self.org.timezone)
```
Comment (norkans7) [inline temba/flows/models.py]: we need this too in Msg.build_message_context

Comment (rowanseymour) [inline temba/flows/models.py]: Good spot... oddly seems there we've just been putting a datetime directly into the context rather than formatting it

**Comment by nicpottier** [comment]
Looks good but coverage gods must be appeased!

File: temba/msgs/models.py [hunk 2]
```diff
@@ -316,9 +316,6 @@ def get_message_delivered_count(self):
     def get_message_failed_count(self):
         return self.get_messages().filter(status__in=[FAILED, RESENT]).count()
 
-    def get_first_message(self):
```
Comment (rowanseymour) [inline temba/msgs/models.py]: Not called anywhere

**Pull request merged**